### ISSUE TITLE
Allow for customization of image badges in wallet addresses

### ISSUE THREAD

**Issue by doubled1c3** [ArkEcosystem/desktop-wallet#1024]
**Is your feature request related to a problem? Please describe.**
I noticed that wallet addresses have unique image badges, but I cannot customize them for a more personable experience

**Describe the solution you'd like**
It would be nice for the user to be able to customize the images for wallet addresses

**Additional context**
It would also probably be a good idea to alert the user that the image is viewable to them only, and won't be uploaded to any server

![image](https://user-images.githubusercontent.com/32085788/51771794-05296a80-209f-11e9-86ef-aeab26c77442.png)

**Comment by j-a-m-l** [comment]
The current implementation is based on a library that works quite well, but they don't have any distinguishable attribute to know that are for ARK. So, we are going to replace those images (identicons) with others that are ARK exclusive.

Although it's almost finished, isn't something that we have been pushing over other tasks, so I guess that it wouldn't be included in our next release.

IMHO, the next step would be adding those identicons to the mobile wallet and explorer, so users could recognize their wallets easily, because it's a lot easier to remember the image than the entire address. This is already done in Ethereum and Nimiq (and their third party applications), and provides the security that you're seeing the right address without reviewing each character of the address.

So, choosing the image of each wallet address, may be a good idea, but it would make them less easy to recognize outside the desktop wallet.

**Comment by doubled1c3** [comment]
I created this github issue as an example, [for this turorial video I put together](https://youtu.be/NvNz-JmwE7o). To clarify, overriding the identicon with a custom image would just be for the desktop wallet single machine. 

While the new upgrade you described is pretty cool, it would also be cool (as an added optional feature) to be able to replace it with an image that is personally significant to the user, like for example, if they want to put an image of a palm tree on one of their addresses for a "vacation fund" or something similar. 

It's an auxiliary feature that may never even happen, but I just chose it as the example for the video. Naturally, the new identicons could have a library of common images that are network-wide (like for example, ARK's version of a palm tree) to increase personalization. Cheers @j-a-m-l
